Makefile

```diff
@@ -61,7 +61,7 @@ clean:
 	rm -f skopeo docs/*.1
 
 install: install-binary install-docs install-completions
-	install -D -m 644 default-policy.json ${CONTAINERSSYSCONFIGDIR}/policy.json
+	install -D -m 644 default-trust.json ${CONTAINERSSYSCONFIGDIR}/trust.json
```

Comment:
This policy path is hard-coded inside `containers/image`, needs updating there as well.

(As can be seen by this PR not changing the path of anything consuming the image.)